### PR TITLE
frontend: Add PolygonZKEvm support

### LINKED ISSUE #1775: Support Polygon ZKEVM on frontend
# Rationale
We want user to be able to bridge to and from Polygon ZKEVM via our frontend for this we need to :
# Implementation

- [ ] Add chainconfig and tokens are added to the file

### PR DESCRIPTION
**Pull request by GabrielBuragev**
Closes #1775 

Also, additionally adds a way to hide certain networks before we officially announce/launch. 
This means that once we are ready to announce Polygon ZKEvm support, we just need to flip the value of the boolean flag `hidden` (or remove it completely) inside the chain config file.

### CODE REVIEW

File: frontend/config/chains/1101.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "identifier": 1101,
```
Comment (manuelwedler) [inline frontend/config/chains/1101.json:2]: Don't we also need a config for the polygon goerli testnet?

**Review by manuelwedler — COMMENTED** [review]
Looks good! Just wondering why there is no configuration for goerli.

File: frontend/config/chains/1101.json (hunk 1 above)
Comment (GabrielBuragev) [reply frontend/config/chains/1101.json:2]: Good point. Will add it.

**Review by manuelwedler — APPROVED** [review]
Perfect!

**Pull request merged**